include/revng/PipelineDescriptor/PipelineDescriptor.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/PipelineDescriptor/AnalysisArgument.h"
```

Comment:
1. Push all of this to `revng/Pipeline/Descriptor/`.
2. "Descriptor" is something you "attach" to something, like a label. We can go for `Description` or `Metadata`.

Reply:
1. Done
2. Renamed to `Description`